### ISSUE TITLE
Comparison issue

### ISSUE THREAD

**Issue by toncid** [hexorx/countries#407]
It is impossible to compare a country with nil:

```
2.3.1 :001 >  ISO3166::Country.new('GB') == nil
NoMethodError: undefined method `alpha2' for nil:NilClass
        from /Users/tonci/.rvm/gems/ruby-2.3.1/gems/countries-2.0.2/lib/countries/country.rb:35:in `=='
        from (irb):1
        ...
```

Should be easy to fix.

**Comment by rposborne** [comment]
Patched and released 2.0.3

**Comment by toncid** [comment]
Thanks! This was really quick.
